app/lib/common/dialogs/member_info_drawer.dart

```diff
@@ -1,6 +1,7 @@
 import 'package:acter/common/dialogs/block_user_dialog.dart';
```

Comment:
I see there there are multiple class related to memberInfo features in this single file (`app/lib/common/dialogs/member_info_drawer.dart)`. I would suggest to create `member_info` folder and put different class in different files for better code readability.

Reply:
They are all _intentionally_ internal (prefixed with a `_`). If I put them into several files or folders I would have to make them public and but I don't want any API public other than the `showMemberInfoDrawer()` fn. That's why I will leave it as is.
`